test-suite/exonum-py-tests/suite/common.py

```diff
@@ -0,0 +1,89 @@
+"""Module containing common scenarios that can be used
+for writing tests with less boiler-plate."""
+
+from typing import List, Dict, Any
+import unittest
+
+from suite import ExonumNetwork, ProcessOutput, ProcessExitResult
+
+
+def run_dev_node(application: str) -> ExonumNetwork:
+    """Starts a single node in the run-dev mode and returns
+    `ExonumNetwork object with the running node.
+
+    Example:
+
+    >>> network = run_dev_node("exonum-cryptocurrency-advanced")"""
+    network = ExonumNetwork(application)
+
+    network.run_dev()
+
+    return network
+
+
+def run_n_nodes(application: str, nodes_amount: int) -> ExonumNetwork:
+    """Creates and runs a network with N validators and return an
+    `ExonumNetwork` object with it."""
+
+    address = "127.0.0.1:{}"
+
+    # Assign peer ports starting from 6331.
+    available_peer_port = 6331
+
+    # Assign API ports starting from 8080.
+    available_api_port = 8080
+
+    network = ExonumNetwork(application)
+    network.generate_template(nodes_amount)
+
+    for i in range(nodes_amount):
+        network.generate_config(i, address.format(available_peer_port))
+        available_peer_port += 1
+
+    for i in range(nodes_amount):
+        public_api_address = address.format(available_api_port)
+        private_api_address = address.format(available_api_port + 1)
+        network.finalize(i, public_api_address, private_api_address)
+        available_api_port += 2
+
+    for i in range(nodes_amount):
+        network.run_node(i)
+
+    return network
+
+
+def run_4_nodes(application: str) -> ExonumNetwork:
```

Comment:
Digit in the function name ? Rare case. I'd write a `run_four_nodes`. But not principle.

Reply:
Well, I think that it's shorter, doesn't affect readability and `4` is easier to see that `four`, but I don't insist on it either.